library/general/test/popup_test.rb

```diff
@@ -43,6 +43,35 @@
     end
   end
 
+  describe ".SuppressFeedback" do
+    before do
+      allow(ui).to receive(:BusyCursor)
+      allow(ui).to receive(:GetDisplayInfo).and_return({})
+    end
+
+    it "closes a popup dialog and opens it at the end when feedback is shown" do
+      # twice as first is done by Feedback and second by SuppressFeedback
+      expect(ui).to receive(:OpenDialog).twice
+      expect(ui).to receive(:CloseDialog).twice
```

Comment:
Maybe we should test the correct OpenDialog/CloseDialog/OpenDialog/CloseDialog sequence so we can ensure the correct final state.